packages/smb-onboarder/src/core/app.tsx

```diff
@@ -10,8 +10,8 @@ import UploadProvider from '@/components/providers/upload-provider'
 
 export const globals = css`
   :global() {
-    body {
-      background-color: unset;
+    html {
+      font-size: 14px;
```

Comment:
This is inherited from Elements I think. Anyway, should be 16px unless desktop when it's 14px. See my comment in index.tsx, we should get this for free now